core/new-gui/src/app/workspace/service/drag-drop/drag-drop.service.ts

```diff
@@ -46,6 +46,13 @@ export class DragDropService {
 
   private static readonly DRAG_DROP_TEMP_OPERATOR_TYPE = 'drag-drop-temp-operator-type';
 
+  private readonly operatorClosestPositionStream =  new Subject <{status: boolean, operatorID: string}>();
+  private readonly operatorSuggestionUnhighlightStream =  new Subject <{status: boolean, operatorID: string}>();
+
+  private suggestionOperator: OperatorPredicate | undefined;
+  private mouseAt: Point | undefined;
+  private isLeft: Boolean | undefined;
```

Comment:
remove `mouseAt` and add some descriptions of what `isLeft` and `suggestionOperator` represent.

Reply:
Make `isLeft` default to `false` so that it does not need to be undefined. When you are checking whether there should be a suggestion, `suggestionOperator` is sufficient by checking whether it equals to undefined.